packages/store/migrations/index.ts

```diff
@@ -99,4 +134,15 @@ const migrations: Migration[] = [
 			]);
 		},
 	},
+	{
+		version: 3,
+		description:
+			"This migration add encodedPublicKey to Sui PublicKeyDocument['meta']",
+		scope: 'app',
+		migrate: async () => {},
+	},
 ];
+
+export const getLatestMigrationVersion = () => {
+	return migrations[migrations.length - 1].version;
```

Comment:
get by max value

Reply:
we agreed that migrations will contain all the migrate version, so I think this method still worked in the way we expected